pages/galleri/index.tsx

```diff
@@ -0,0 +1,230 @@
+import Layout, { Metadata } from '../../components/Layout';
+import styled from 'styled-components';
+import { useCallback, useEffect, useState } from 'react';
+import Image from 'next/image';
+import { itdageneBlue } from '../../utils/colors';
+
+// alle scr url's må være i form av /static/...
+// for some fucking reason
+const images = [
+  'DSC_1043.jpg',
+  'DSC_1003.jpg',
+  'DSC_1053.jpg',
+  'DSC_1105.jpg',
+  'DSC_1128.jpg',
+  'DSC_1137.jpg',
+  'DSC_1141.jpg',
+  'DSC_1145.jpg',
```

Comment:
Is the plan to hard code all images in here?

Reply:
I talked it over with Ludvig and we concluded that implementing a smarter solution would be difficult and could be done in a later pr. 

But it is totally in my plans to do so.

Reply:
On another note, the comment here doesn't really make sense?

Reply:
Already removed. Really unprofessional of me to curse 🥲